pinot-core/src/test/java/org/apache/pinot/queries/TextSearchQueriesTest.java

```diff
@@ -569,10 +591,16 @@ public void testTextSearch() throws Exception {
 
     query = "SELECT INT_COL, SKILLS_TEXT_COL FROM MyTable WHERE TEXT_MATCH(SKILLS_TEXT_COL, '\"distributed systems\" Java C++') LIMIT 50000";
     testTextSearchSelectQueryHelper(query, expected.size(), false, expected);
-
     query = "SELECT COUNT(*) FROM MyTable WHERE TEXT_MATCH(SKILLS_TEXT_COL, '\"distributed systems\" Java C++') LIMIT 50000";
     testTextSearchAggregationQueryHelper(query, expected.size());
 
+    // test for the index configured to use AND as the default
+    // conjunction operator
+    query = "SELECT INT_COL, SKILLS_TEXT_COL FROM MyTable WHERE TEXT_MATCH(SKILLS_TEXT_COL_1, '\"distributed systems\" OR Java OR C++') LIMIT 50000";
```

Comment:
Could you add some tests which uses OR operator? If there is some cases like that, can we add some comments on that and mention the test with `AND` in this test class?

Reply:
There are plenty of existing tests that use both and (explicitly) and or (both implicitly and explicitly).

In addition, I have added new tests to test the behavior when the config is set to true. In this case also, I have added tests for both and and or implicitly and explicitly